hw_19/js/script.js

```diff
@@ -0,0 +1,77 @@
+const array = [16, -37, 54, -4, 72, -56, 47, 4, -16, 25, -37, 46, 4, -51, 27, -63, 4, -54, 76, -4, 12, -35, 4, 47];
+console.log(array);
+
+// 1. Знайти суму та кількість позитивних елементів.
+
+const positive = array.filter(value => value >= 0);
+const numPos = positive.length;
+const sum = positive.reduce((accumulator, value) => accumulator + value);
+
+console.log(`1. There are ${numPos} positive elements in the array. The sum of these elements equals ${sum}`);
+
+// 2. Знайти мінімальний елемент масиву та його порядковий номер.
+
+const arraySort = [...array].sort((a, b) => a - b);
+console.log(arraySort);
+
+const minVal = arraySort[0];
+
+const minPosition = array.indexOf(minVal) + 1;
+console.log(`2. The minimal number is ${minVal}. It's position is ${minPosition}`);
```

Comment:
very complex solution. It will be better to use one loop where you can get min value and index